src/SettingsView/EventType.js

```diff
@@ -51,7 +51,7 @@ class EventType extends ConnectionComponent {
 	
 	render() {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This function has 101 lines, which is greater than the 100 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS138)